### PR TITLE
Feat/starting league statistics methods

### PR DESCRIPTION
**Pull request by JaredMHobson**

### CODE REVIEW

File: spec/stat_generator_spec.rb [hunk 1]
```diff
@@ -22,4 +22,72 @@
...
+
+      expect(best_team).to eq("Reign FC")
+    end
+  end
+
+  describe "#worst_offense" do
+    it "can return the name of the team with the lowest average number of goals scored per game across all seasons" do
+      worst_team = @stat_generator.worst_offense
+
+      expect(worst_team).to eq("Utah Royals FC")
```
Comment (RodrigoACG) [inline spec/stat_generator_spec.rb:90]: Good job on getting to three methods!!

File: lib/game_team.rb [hunk 2]
```diff
@@ -1,6 +1,10 @@
 class GameTeam
+  attr_reader :team_id, :goals
+
   def initialize(details)
     @game_id = details[:game_id]
+    @team_id = details[:team_id].to_i
+    @goals = details[:goals].to_i
```
Comment (NeilTheSeal) [inline lib/game_team.rb:7]: This might be better converted to a symbol rather than a string. Let's debug this later

File: lib/team.rb [hunk 3]
```diff
@@ -1,6 +1,9 @@
 class Team
+  attr_reader :team_id, :team_name
+
   def initialize(details)
-    @team_id = details[:team_id]
+    @team_id = details[:team_id].to_i
+    @team_name = details[:teamname]
   end
```
Comment (NeilTheSeal) [inline lib/team.rb:7]: Same thing with @team_id. We will need to figure out a consistent data type that works for all of us

**Review by NeilTheSeal — APPROVED** [review]
Great job! You're the best coder in history!

**Pull request merged**